src/fragments/lib/troubleshooting/common/upgrading.mdx

```diff
@@ -177,7 +177,7 @@ Wherever you called `Amplify.configure({ aws-exports });` previously (usually in
 </Columns>
 
 <Callout>
-  `Amplify.configure()` will now accept either the config JSON file or a strongly typed configuration object. Therefore, if you need to add additional configuration, you will call configure twice: once with the contents of **amplifyconfiguration.json**, and then again using `Amplify.getConfig()` plus any additions. Keep in mind that any call to configuration will fully override previous configurations, so pay special attention to nested configurations.
+  `Amplify.configure()` accepts either the config JSON file or a `ResourcesConfig` configuration object. You can use the `parseAmplifyConfig` util to parse the contents of the config JSON file **amplifyconfiguration.json** into the `ResourcesConfig` configuration object and override properties as needed. Keep in mind that any call to configuration will fully override previous configurations, so pay special attention to nested configurations.
```

Comment:
do we have ResourcesConfig documented anywhere that we could link out to?

Reply:
Makes sense, updated to link ResourcesConfig to [Configure Amplify categories](https://docs.amplify.aws/gen1/javascript/tools/libraries/configure-categories/)